src/SDKs/DataMigration/Management.DataMigration/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd datamigration/resource-manager %*
```

Comment:
Code must be generated using the generate.ps1, please remove this file

Reply:
Done.